fragments/tap-workload/accelerator.axl

```diff
@@ -0,0 +1,12 @@
+engine {
+  Exclude({"README.md"})
+  if(#bsGitRepository > "") {
+    let repoUrl = T(org.springframework.web.util.UriComponentsBuilder).fromUriString('https://' + #bsGitRepository).build(true) in
+    {
+      applyTo("**/workload.yaml" || "**/workload-native.yaml" || "**/workloads.yaml" ) {
+        ReplaceText(regex: {pattern: "(?m)url: .*$", with: 'url: ' + #repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]})
```

Comment:
Why go to regex instead of OpenRewriteRecipe?

Reply:
Much cheaper in terms of processing time, not sure why we had one recipe and one regex originally, should be the same for both I would think